resources/js/Components/Detail/CampaignDetail/CampaignDetailBox.tsx

```diff
@@ -13,59 +18,108 @@ const StyledTypography = styled(Typography)({
 
 type CampaignDetailBoxPropType = {
     campaign: CampaignData
+    userCharacter: UserCharacterData
+    setIsEditDrawerOpen: (payload: boolean) => void
 }
 
-const CampaignDetailBox = () => (
-    // {campaign}: CampaignDetailBoxPropType
-    <Box sx={{p: 5, backgroundColor: 'primary'}}>
-        <Grid container columnSpacing={1} rowSpacing={6} xs={6}>
-            <Grid item xs={12}>
-                <StyledTypography>NAME</StyledTypography>
-                <Typography>
-                    {/* {campaign.title} */}
-                    Campaign 1
-                </Typography>
-            </Grid>
-            <Grid item xs={12}>
-                <StyledTypography>SESSIONS PLAYED</StyledTypography>
-                <Typography>
-                    {/* {campaign.sessions_played} */}
-                    25
-                </Typography>
-            </Grid>
-            <Grid item xs={12}>
-                <StyledTypography>CHARACTERS</StyledTypography>
-                <Typography>
-                    Character 1, Character 27, Character 31, Character 45
-                </Typography>
-            </Grid>
-            <Grid item>
-                <Stack spacing={3} direction='row'>
-                    <Button
-                        variant='contained'
-                        startIcon={<CreateIcon fontSize='small' />}>
-                        INVITE
-                    </Button>
-                    <Button
-                        variant='contained'
-                        startIcon={<CreateIcon fontSize='small' />}>
-                        UPDATE
-                    </Button>
-                    <Button
-                        variant='contained'
-                        startIcon={<HistoryEduIcon fontSize='small' />}>
-                        ENTRY
-                    </Button>
-                    <Button
-                        variant='contained'
-                        startIcon={<AutoStoriesIcon fontSize='small' />}>
-                        DM Entry
-                    </Button>
-                </Stack>
-            </Grid>
-        </Grid>
-    </Box>
-)
+const CampaignDetailBox = ({
+    campaign,
+    userCharacter,
+    setIsEditDrawerOpen,
+}: CampaignDetailBoxPropType) => {
+    const {t} = useTranslation()
+    const [openInviteSnackbar, setOpenInviteSnackbar] = useState(false)
+
+    const handleClick = () => {
+        navigator.clipboard.writeText(campaign.code)
+        setOpenInviteSnackbar(true)
+    }
+
+    const handleClose = () => {
```

Comment:
same for this one, btw if it's only one statement there is no need to do 
```
= () => {
     someFunction()
}
```
do this instead
```
= () => someFunction()
```